services/QuillLessons/app/components/classroomLessons/teach/currentSlide.tsx

```diff
@@ -319,7 +319,7 @@ class CurrentSlide extends React.Component<CurrentSlideProps & StateFromProps, a
               onlyShowHeaders={this.props.classroomSessions.onlyShowHeaders}
               updateToggledHeaderCount={this.updateToggledHeaderCount}
               saveModel={this.saveModel}
-              clearStudentSubmission={this.clearStudentSubmission}
+              clearStudentSubmission={this.clearStudentSubmission.bind(this)}
```

Comment:
Crazy that this is all that was wrong! Gotta love being able to test locally. Can we move the binding to the constructor method to fit the pattern?

Reply:
Good call.  I've updated the PR to do so.